DEFCON Z/Assets/Scripts/Units/Special/PrecisionBomb.cs

```diff
@@ -12,7 +12,7 @@ public class PrecisionBomb : SpecialAbility
         public float planeSpeed;
         public float bombSpeed;
         public float bombDamage;
-        public static string abilityName = "Air Strike";
+        public new static string abilityName = "Air Strike";
```

Comment:
This need not be overriden by new. Since the base class `SpecialAbility` has the property already. Just move the code into the initialization.